app/experimenter/nimbus-ui/src/lib/experiment.ts

```diff
@@ -44,15 +39,13 @@ export type StatusCheck = ReturnType<typeof getStatus>;
 
 // Common redirects used on all Edit page components
 export function editCommonRedirects({ status }: RedirectCheck) {
-  // If experiment is in review or preview, it can't be edit,
-  // but it's also not locked, so send you to the review page
-  if (status.review || status.preview) {
-    return "request-review";
+  // If experiment is released, send you to the summary page
+  if (status.released) {
+    return "";
   }
 
-  // If experiment is locked (like in review, complete),
-  // send you to the summary page (slug root)
-  if (status.locked) {
-    return "";
+  // User still need to see the 'edit' sidebar but cannot be edited
+  if (!status.idle || status.preview) {
```

Comment:
Same here

Reply:
This PR already reduces the "extra" FE statuses from 3 (released, locked, preparation) to 1 (released), I think this is a good idea for sure because I like the BE managing these statuses, but maybe we should do that in a follow up? We're not really adding new logic here, just editing old logic. Maybe there's other convenience flags we could do at the same time. 🤔

Reply:
Yeah I like isEditable ([something like this?](https://gist.github.com/jodyheavener/9c3608cddad0af2f22d0bbb07646d8f4)), might be a good opportunity to add other boolean properties that check on combo statuses. But agree that it might be better suited to a followup?